### PR TITLE
Polarized STAR and PHENIX A_L W 

### PR DESCRIPTION
**Pull request by giacomomagni**
Implementation of the polarized asymmetries DY data from RHIC.

**STAR datasets:**

| Dataset  | Obs.  |  Status  |   Comments  | Correlations | Inspire | HepData |
|---|---|---|---|---|---|---|
| STAR_WMWP_510GEV |  STAR_WMWP_510GEV_WM-AL STAR_WMWP_510GEV_WP-AL  | ✅  | Figure 5 provides the combination with the 2011-2012 data |  Table1 of the Appendix for 2013 run only, not implemented |  [paper](https://arxiv.org/abs/1812.04817)  |  [dataset](https://www.hepdata.net/record/ins1708793) |
| | | ❌ | Included in the combination with 2013 data | |  [paper](https://inspirehep.net/literature/1292792)  |  [dataset](https://www.hepdata.net/record/ins1292792) |

**PHENIX datasets:**

| Dataset  | Obs.  |  Status  |   Comments  | Correlations | Inspire | HepData |
|---|---|---|---|---|---|---|
| | $A_L$ in $W^\pm +Z$ | ❌ | see below | |  [paper](https://inspirehep.net/literature/1365091)  |  [dataset](https://www.hepdata.net/record/ins1365091) |
| | $A_L$ in $W^\pm +Z$ | ❌ | see below | |  [paper](https://inspirehep.net/literature/1667398)  |  [dataset](https://www.hepdata.net/record/ins1667398) |

### CODE REVIEW

**Comment by giacomomagni** [comment]
Regarding the Phenix measurement we might have a problem. 
My understanding is that Phenix has measured $A_L$ in $W^\pm+Z$ process, 
despite what they say in the paper title. 
In particular I read from [here](https://arxiv.org/pdf/1504.07451):

```
Figure 3 shows the combined asymmetry for all of the
PHENIX data sets and published data from STAR [11].
The two data sets cannot be compared directly, because
PHENIX measures the asymmetry from W± +Z decays,
while the STAR result is solely from W± decays. The
comparison can be made through the curves, which account for the specifics of each measurement.
```

So the numbers that are provided in HepData are not exactly for the same quantity measured by Star. 
You can then see a difference of the predictions for the two processes in Figure 3. 

@Radonirinaunimi , @enocera do we have a code to produce sensible predictions also for Phenix ?
If not, do we want to have them implemented for the future anyway?

File: nnpdf_data/nnpdf_data/new_commondata/STAR_WMWP_510GEV/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  nnpdf31_process: "DY CC"
+  experiment: "STAR"
+
+implemented_observables:
+  - observable_name: "WM-AL"
+    observable:
+      description: "Longitudinal single-spin asymmetry $A_L$ for $W^-$ production as a function of the lepton pseudorapidity, $\eta_e$"
+      label: "$A_L$"
+      units: ""
+    process_type: "EWK_RAP"
```
Comment (Radonirinaunimi) [inline nnpdf_data/nnpdf_data/new_commondata/STAR_WMWP_510GEV/metadata.yaml:25]:
```suggestion
    process_type: "EWK_RAP_ASY"
```

File: nnpdf_data/nnpdf_data/new_commondata/STAR_WMWP_510GEV/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        - - STAR_WMWP_510GEV_WM-AL-POL
+        - - STAR_WMWP_510GEV_WM-AL-UNPOL
+      operation: "ratio"
+
+  - observable_name: "WP-AL"
+    observable:
+      description: "Longitudinal single-spin asymmetry $A_L$ for $W^+$ production as a function of the lepton pseudorapidity, $\eta_e$"
+      label: "$A_L$"
+      units: ""
+    process_type: "EWK_RAP"
```
Comment (Radonirinaunimi) [inline nnpdf_data/nnpdf_data/new_commondata/STAR_WMWP_510GEV/metadata.yaml:62]:
```suggestion
    process_type: "EWK_RAP_ASY"
```

**Review by Radonirinaunimi — COMMENTED** [review]
To allow for these asymmetries to have negative pseudodata.

**Comment by giacomomagni** [comment]
Looking here:

https://agenda.infn.it/event/38132/contributions/234387/attachments/120751/175847/Trieste-wv.pdf

It seems that our friends do include the 3 datapoints from PHENIX as well in their $A_L$ ($W^+$) datasets...

**Comment by Radonirinaunimi** [comment]
> Looking here:
> 
> https://agenda.infn.it/event/38132/contributions/234387/attachments/120751/175847/Trieste-wv.pdf
> 
> It seems that our friends do include the 3 datapoints from PHENIX as well in their AL (W+) datasets...

Interesting! I am curious to know how do they do then? Maybe they just don't care about the distinction $W^\pm$ vs $W^\pm \otimes Z$...

**Pull request merged**